benchmarking/mps_fixed_to_free.awk

```diff
@@ -0,0 +1,23 @@
+function fixname(s,n,t) {
```

Comment:
The script should have a license header. Could you also add a top-level comment explaining what it does?

Reply:
Done.